spec/requests/api/providers_spec.rb

```diff
@@ -254,8 +254,8 @@
     it "supports multiple resource edits" do
       api_basic_authorize collection_action_identifier(:providers, :edit)
 
-      p1 = FactoryGirl.create(:ext_management_system, :name => "name1")
-      p2 = FactoryGirl.create(:ext_management_system, :name => "name2")
+      p1 = FactoryGirl.create(:ems_redhat, :name => "name1")
+      p2 = FactoryGirl.create(:ems_redhat, :name => "name2")
```

Comment:
After first commit, we now serialize more than data than before. For instance virtual_attributes. Previously, we have just saw `nil` for each attempt to read a virtual_attribute, now we call method to get it serialized.

This particular test failed because of this. That is because `emstype_description` does not work on stock `ext_management_system`.